esp/scm/ws_fs.ecm

```diff
@@ -540,10 +540,7 @@ ESPresponse [exceptions_inline] FileListResponse
 ESPrequest [nil_remove] DropZoneFileSearchRequest
 {
     string DropZoneName;
-    string FileName;
```

Comment:
This method was added to 6.0.0 several days ago. It has not been used by any one. And we haven't done any build for 6.0.0. I think it should be no compatibility concern.